src/System.Security.Permissions/src/System/ServiceProcess/ServiceControllerPermission.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security.Permissions;
+
+namespace System.ServiceProcess
+{
+    public sealed class ServiceControllerPermission : ResourcePermissionBase
+    {
+        public ServiceControllerPermission() { }
+        public ServiceControllerPermission(PermissionState state) : base(state) { }
+        public ServiceControllerPermission(ServiceControllerPermissionAccess permissionAccess, string machineName, string serviceName) { }
+        public ServiceControllerPermission(ServiceControllerPermissionEntry[] permissionAccessEntries) { }
+        public ServiceControllerPermissionEntryCollection PermissionEntries { get; }
+        internal void AddPermissionAccess(ServiceControllerPermissionEntry entry) { }
```

Comment:
The internal and private methods should not be needed.

Reply:
(multiple places throughout the PR)

Reply:
can we remove protected too ?

Reply:
No protected are externally visible. (Note they are in the ref/* files)